README.md

```diff
@@ -33,15 +33,15 @@ Antrea has been tested with Kubernetes clusters running version 1.16 or later.
 ## Getting Started
 
 Getting started with Antrea is very simple, and takes only a few minutes.
-See how it's done in the [Getting started](docs/getting-started.md) document.
+See how it's done in the [Getting started](/docs/getting-started.md) document.
```

Comment:
You meant to remove "/" from "/docs/getting-started.md"?

Reply:
Yes. These links should be relative. We are working on a way to apply a specific transformation for these links in the site rendering. For now, we have to copy this file and remove `docs/` prefix as well since the rendering of the Documents section of the site renders all files from the same directory.

Reply:
I got that. But here your change actually added "/" back, instead of removing it.